src/main/java/mServer/tool/MserverXmlLesen.java

```diff
@@ -76,7 +71,6 @@ public static void xmlDatenLesen(String datei) {
         } catch (Exception ex) {
             MserverLog.fehlerMeldung(909078531, MserverXmlLesen.class.getName(), "xmlDatenLesen", ex);
         } finally {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00108)

Reply:
No change, so write a new Task for refactoring.